### PR TITLE
Add comment

### PR DESCRIPTION
**Pull request by kaisdoss**

### CODE REVIEW

File: back/models/product.js [hunk 1]
```diff
@@ -2,9 +2,16 @@ const mongoose = require("mongoose");
...
 const ProductSchema = mongoose.Schema({
   name: { type: String, require: true },
-  Number: { type: Number, require: true },
+  number: { type: Number, require: true },
+  etat: {
+    type: String,
+    enum: ["en stock", "hors stock"],
+    default: "en stock",
+  },
+
```
Comment (SeifESlimene) [inline back/models/product.js:11]: Why this space?

File: back/models/product.js [hunk 2]
```diff
@@ -2,9 +2,16 @@ const mongoose = require("mongoose");
...
+  buyingPrice: { type: Number },
```
Comment (SeifESlimene) [inline back/models/product.js:12]: why not required like others?

File: back/routes/productRoute.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
+const express = require("express");
+const router = express.Router();
+const Product = require("../models/product");
+const authMiddleware = require("../helpers/authMiddleware");
+
+// Route Create New product
+// Path : http://localhost:3000/addProduct
+router.post("/", authMiddleware, (req, res) => {
```
Comment (SeifESlimene) [inline back/routes/productRoute.js:8]: Please careful!

**Review by SeifESlimene — COMMENTED** [review]
Please see my comments and correct them!

File: back/models/product.js (hunk 1 above)
Comment (kaisdoss) [reply back/models/product.js]: done

File: back/models/product.js (hunk 2 above)
Comment (kaisdoss) [reply back/models/product.js]: done

File: back/routes/productRoute.js (hunk 3 above)
Comment (kaisdoss) [reply back/routes/productRoute.js]: ok 😉

**Pull request merged**